ibm_catalog.json

```diff
@@ -247,11 +222,11 @@
               "key": "key_type",
               "options": [
                 {
-                  "displayname": "rsa",
+                  "displayname": "Rsa",
```

Comment:
Display name should be `RSA`
<img width="223" height="247" alt="image" src="https://github.com/user-attachments/assets/2843acce-aedc-49c1-b4d1-8ac1e1034f71" />

https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-prepare-create-certificates#choose-key-algorithm